### ISSUE TITLE
Add an option to add a game to an existing glossi shortcut

### ISSUE THREAD

**Issue by RamazanKara** [LemmusLemmus/GlosSI-Integration-Playnite#7]
Example Scenario: Adding new Switch Games to an existing Switch Controller Config.

**Comment by LemmusLemmus** [comment]
Glad to hear from you! Just to make sure, have I interpreted the feature correctly? <sup><sub>I hope my attempt at describing my interpretation is not too unclear...</sub></sup>

Let's say I have a Switch game called "Game A". I make a GlosSI configuration for it via GlosSIConfig.exe and then add it to Steam as a Steam shortcut. Afterwards, I want to use the same shortcut with this Playnite extension instead. Is the feature then to be able to use the already created Steam shortcut for "Game A" (with its controller configuration and potentially also its associated GlosSI configuration) in Playnite, instead of creating a new Steam shortcut and GlosSI configuration?

Or do you perhaps want to be able to copy the Steam controller configuration of "Game A" when adding new games?

**Comment by RamazanKara** [comment]
Hi Lemmus, thank you for your quick reply. 

Lets say I have multiple Emulators, games that can use trackpad mouse or games that need to map the right analogue stick to the track Pad. 

With this feature, I could for example instead of making tons of configurations for each of my games, I could just set up configs per Emulator or per Game Type and that would save a lot of time, especially if you are somebody who has tons of emulated games.

So basically a sort of grouping function or the ability to have the add on use one shortcut that it created for multiple games. 

And sorry for my inaccurate wording on my first comment as it was quite unclear.

**Comment by LemmusLemmus** [comment]
Oh, I see! That would make sense as an option. It should _hopefully_ be possible to automatically apply Steam Controller configurations, but no promises.

Would this kind of implementation make sense? Any suggestions?
- --
In the add-on settings menu:
**Defaults used when creating new shortcuts:**
-- Field drop-down list _(What kind of field(s) should one be able to choose from? "Emulator profile" maybe? "Source" maybe? "Platform" maybe? In this drop-down list there should probably also be a default option.)_
-- Choose one of the pre-existing Steam Controller Configurations or none.
-- Edit the GlosSI Configuration used, or use the default one. _(maybe? I don't really see as of now why you would want to use different GlosSI configurations for different games, but such a feature wouldn't hurt I guess.)_
- --

In the mean time, in the Steam overlay, you can save your Steam controller configuration as a template and then apply said template to any game (in-game). This cannot be done automatically, but at least it is a lot less tedious than creating a new controller configuration every time.
**EDIT:** Apparently there are quicker methods to go about doing this manually, that do not require saving the configuration as a template.

**Comment by RamazanKara** [comment]
Not sure what the best approach would be, but just right clicking on game/s - and a menu called "add to existing steam glossi shortcut." would be enough. But your idea for defaults based on filters is also really excellent and a way better approach.

This would come in really handy with emulators. We have some Emulators with 1000s of games, where you would previously have to map all to one steam controller template on the steam overlay like you described (e.g. each emulator has different shortcuts, different button settings etc.) this way it would make it really easy to manage these.
